app/assets/javascripts/pages/EligibilityEstimatorController.js.coffee

```diff
@@ -2,20 +2,24 @@
 ###################################### CONTROLLER ##########################################
 ############################################################################################
 
-EligibilityEstimatorController = ($scope, $state) ->
-  formDefaults =
-    'household_size': ''
-    'income_timeframe': ''
-    'income_total': ''
+EligibilityEstimatorController = ($scope, $state, ListingService, IncomeCalculatorService) ->
+  $scope.filter_defaults = ListingService.eligibility_filter_defaults
+  $scope.filters = angular.copy(ListingService.eligibility_filters)
+
+  # check if we've used the IncomeCalculator to calculate income
+  if IncomeCalculatorService.calculateTotalYearlyIncome() > 0
```

Comment:
@kaptron this looks interesting, I'm usually used to seeing everything encompassed under $scope, how does this work in terms of "refreshing"? Is it always constantly called since it starts with a condition?